migrations/20201004075522-add-organization.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.createTable('organzation', {
```

Comment:
organization is spelled wrong lol
let's use plural form when creating tables
so, change to `organizations`